modules/sugar/src/test/ts/browser/ScrollTest.ts

```diff
@@ -127,7 +127,9 @@ UnitTest.asynctest('ScrollTest', (success, failure) => {
     const cX = Math.round(center.left);
     const cY = Math.round(center.top);
 
-    const noVisibleScrollbarBrowser = platform.os.isMacOS() || (platform.browser.isFirefox() && platform.os.isLinux()) || (platform.browser.isFirefox() && platform.os.isWindows() && platform.os.version.major >= 11);
+    // TINY-9203: due to Win11 FF adopting native hidden scrollbar behavior and current inability to distinguish between Win10 and Win11
```

Comment:
Just make sure the reasoning is in the associated JIRA ticket. Maybe even link in the JIRA ticket to any slack discussions.